runtime/msg.h

```diff
@@ -158,6 +158,8 @@ struct msg {
 /* rawmsg does not include a PRI (Solaris!), but PRI is already set correctly in the msg object */
 #define PRESERVE_CASE	0x200
 /* preserve case in fromhost */
+#define KEEP_FORMAT     0x400
```

Comment:
we need to discuss the need for this - so far, I do not understand why it is useful

Reply:
This flags value allow an input module to inform the output module that this message message should be send as is (rawmsg) instead of using a template. (The json with system metrics as we discuss ealier for example). The core doesn't need to be aware of this and should just transmit it with the message.

Reply:
This flag actually deeply concerns me -- it  looks like it breaks layers. I do not really understand why it is needed. You can always put in proper templates, so why add a special way to access `rawmsg` property?

Reply:
suggestion: mabye we should remove it from current PR and add it to the omqmpq updates? In context it may be easier to discuss and we remove one show stopper from this PR here?